README.md

```diff
@@ -43,3 +43,5 @@ npx openapi-typescript-codegen --input http://localhost:8000/openapi.json --outp
 
 - [FastAPI](https://fastapi.tiangolo.com/python-types/)
 - [openapi-typescript-codegen](https://github.com/ferdikoomen/openapi-typescript-codegen)
+
+Test
```

Comment:
Consider providing more context or useful information instead of just adding the word "Test". [important]